### DISCUSSION TITLE
"not-muted symbol" instead of "muted symbol"

### DISCUSSION THREAD

**Discussion by Elementari** [mbnuqw/sidebery#888] [Ideas]
Hello, 
I've often had the problem that I'm wearing headphones to listen to spotify, and I sometimes change the volume within the app, and sometimes global volume for MacOS. Sooner or later I will be listening to the news or so in a Firefox tab, and the volume will be very loud, hurting my ears. Another situation also seems to be for people who are browsing in an office (with other co-workers around), with loud noise suddenly coming from some browser tab. 
A remedy seems to be the extension "Webseiten standardmäßig stummschalten", which is German for "mute tabs by default".
There's also a keyboard shortcut to mute/unmute a tab, which comes handy.
As a result, ALL tabs have the crossed-out musical note symbol, wasting a lot of horizontal space. 
Would the programmers of sideberry consider adding an option that will invert the function and showing a NOT-crossed out note for those tabs that do have sound? 

Moreover, I noticed the muted symbol overlays the favicons for pinned tabs, but not for the lap list. It would be cool if there were an option to make the overlay a global choice, as it would save horizontal space. 

Thank you very much for considering these suggestions!

**Comment by Blitzkowicz** [comment]
These seem like good suggestions. I would also like the option to choose what side of the tab the audio button stays on. With an auto-hide hovering theme, it takes up a lot of space from the tab text when the bar is on the left side.
